component/pom.xml

```diff
@@ -134,10 +156,13 @@
                             org.nd4j.*,
                             com.rits.cloning.*,
                             org.bytedeco.*,
-                            org.objenesis.*
+                            org.objenesis.*,
+                            org.apache.commons.*
+                            com.github.os72.protobuf351.*
                         </Private-Package>
                         <Include-Resource>
-                            META-INF=target/classes/META-INF
+                            META-INF=target/classes/META-INF,
+                            ${project.build.directory}/dependencies/nd4j-native/META-INF/services
```

Comment:
Can you update this to 
META-INF/services=${project.build.directory}/dependencies/nd4j-native/META-INF/services

Reply:
done